mkdocs.yml

```diff
@@ -14,6 +14,7 @@ nav:
       API concepts: concepts.md
       API resources: resources.md
       Security Model: security-model.md
+      API guidelines: guidelines.md
```

Comment:
Nit: Your page title is "Implementation Guidelines". I think that may make the most sense here since this part of the docs is primarily targeted towards implementers.

Reply:
I had a split mind when writing this and it showed up in the patch as well.
I've changed this to `Implementation Guidelines`.